submissions/Roka20012/memory-pair-game/js/script.js

```diff
@@ -0,0 +1,112 @@
+const delayForAnswer = 1000;
+const delay = 500;
+const twoCards = 2;
+const winGame = 12;
+
+let flipper = document.querySelector(".flipper");
+let allCards = document.querySelectorAll(".flipper > div");
+
+function showCards() {
+    setTimeout(function() {
+        allCards.forEach(function(el) {
+            el.classList.toggle("check");
+        });
+    }, delay);
+
+    setTimeout(function() {
+        allCards.forEach(function(el) {
+            el.classList.toggle("check");
```

Comment:
Actually it isn't a very good pattern use a lot of anonymous  functions(but you can). But. In this case you repeat yourself, so you can move this function(el)... to separate function. And use es6 arrow function for anonymous. Have you read about it? `() => {}`

Reply:
Thanks, always forget about that:)